influxdb/src/main/java/com/hazelcast/jet/influxdb/InfluxDbSources.java

```diff
@@ -264,7 +264,7 @@ private InfluxDbSources() {
         checkTrue(connectionSupplier != null, "username cannot be null");
         checkTrue(clazz != null, "clazz cannot be null");
 
-        return SourceBuilder.stream("influxdb-" + database,
+        return SourceBuilder.timestampedStream("influxdb-" + database,
```

Comment:
when we changed it to timestampedStream we also need to provide the timestamp for the measurements when they're emitted.

Reply:
I see, I was looking into whether there is a standard way to retrieve timestamp for each item from InfluxDB.